pkgs/development/compilers/dotnet/build-dotnet.nix

```diff
@@ -121,6 +122,24 @@ stdenv.mkDerivation (finalAttrs: rec {
       "aarch64-darwin" = "osx-arm64";
     };
 
+    updateScript =
+      if type != "sdk" then
+        lib.warn "${pname}-${version}: only the SDK package can be updated - this script will do nothing!"
+        writeShellScript "dummy-update" ''
+          echo "Doing nothing..."
+          echo "Run the updateScript from the SDK package"
```

Comment:
Any reason we don't just throw with this message?

Reply:
If I jave a throw tuere, running the `maintainers/scripts/update.nix` will fail if I update all packages I maintain (since I'm also a maintainer of non-sdk packages). Doibg it like that allows the script to work.